lib/cisco_node_utils/node.rb

```diff
@@ -111,6 +111,8 @@ def massage(value, ref)
           value = false
         elsif /^no / =~ value
           value = false
+        elsif /disable$/ =~ value
+          value = false
```

Comment:
yeah because otherwise the default key does not gets massaged to true instead of false:

D, [2015-11-19T09:07:36.910123 #26876] DEBUG -- : Massaging '["bgp log neighbor changes disable"]' (["bgp log neighbor changes disable"])
D, [2015-11-19T09:07:36.910166 #26876] DEBUG -- : Massaged to 'true'
TestRouterBgp#test_log_neighbor_changes_default = 8.22 s = F

Finished in 8.219810s, 0.1217 runs/s, 0.1217 assertions/s.

  1) Failure:
TestRouterBgp#test_log_neighbor_changes_default [test_router_bgp.rb:623]:
Failed refutation, no message given